Code/Tools/TestImpactFramework/Runtime/Common/Code/Include/Static/Target/Common/TestImpactTarget.h

```diff
@@ -22,6 +22,9 @@ namespace TestImpact
         //! Returns the build target name.
         const AZStd::string& GetName() const;
 
+        //! Returns the target type at runtime.
+        TargetType GetType() const;
```

Comment:
You should probably mark this function `[[nodiscard]]` to make sure the value isn't inadvertently ignore

Reply:
Fixed 👍